src/components/menuItemPropTypes.js

```diff
@@ -154,6 +154,12 @@ const propTypes = {
     /** Should render component on the right */
     shouldShowRightComponent: PropTypes.bool,
 
+    /** Should render title without tooltip or not, we don't show tooltip on some items like, workspace and few more */
+    shouldUseFullTitle: PropTypes.bool,
+
+    /** Render title with tooltips for menu items like, asignee & share somewhere menu */
```

Comment:
I think this needs an update as this sounds like it's a flag to turn on/off rendering.